libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -176,6 +179,10 @@ const AP_Param::GroupInfo AP_ADSB::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("OPTIONS",  15, AP_ADSB, _options, 0),
 
+    // @Group: 
```

Comment:
This group with a empty name does not get picked up by the param docs. Works fine in AP tho.